imqsauth/http.go

```diff
@@ -419,13 +419,18 @@ func httpLoginYellowfin(central *ImqsCentral, w http.ResponseWriter, r *httpRequ
 	}
 	yfGroup := makeYellowfinGroup(permList)
 	if yfGroup != YellowfinGroupNone {
-		cookies, err := central.Yellowfin.LoginAndUpdateGroup(identity, yfGroup)
+		currentModule := ""
+		filter, err := ioutil.ReadAll(r.http.Body)
+		if err == nil {
```

Comment:
You *must* treat all errors with the utmost respect. `ioutil.ReadAll` can fail due to a network disconnect. So it's allowable to ignore certain special errors that you expect to find in practice, but what you're doing here is to ignore all errors. Instead, if there is an error, return immediately. Perhaps an expected error would be io.EOF, but you'll need to test to discover that.